### PR TITLE
Lock execjs to 2.7.0

### PR DESCRIPTION
**Pull request by henrique-ft**
This PR solves the problem of https://github.com/middleman/middleman/issues/2564 

Many people having this problem did the same, it is a `execjs` issue: https://github.com/rails/execjs/issues/99

```
[2022-09-11 13:53:52] ERROR ExecJS::ProgramError: TypeError: Cannot read properties of undefined (reading 'version')
	eval (eval at <anonymous> ((execjs):1:213), <anonymous>:1:10)
	(execjs):1:213
	(execjs):19:14
	(execjs):1:40
	Object.<anonymous> ((execjs):1:58)
	Module._compile (node:internal/modules/cjs/loader:1120:14)
	Module._extensions..js (node:internal/modules/cjs/loader:1174:10)
	Module.load (node:internal/modules/cjs/loader:998:32)
	Module._load (node:internal/modules/cjs/loader:839:12)
	Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/execjs-2.8.1/lib/execjs/external_runtime.rb:39:in `exec'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/execjs-2.8.1/lib/execjs/external_runtime.rb:21:in `eval'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/execjs-2.8.1/lib/execjs/runtime.rb:64:in `eval'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/autoprefixer-rails-9.8.6.5/lib/autoprefixer-rails/processor.rb:170:in `runtime'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/autoprefixer-rails-9.8.6.5/lib/autoprefixer-rails/processor.rb:53:in `process'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/middleman-autoprefixer-2.10.1/lib/middleman-autoprefixer/extension.rb:83:in `prefix'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/middleman-autoprefixer-2.10.1/lib/middleman-autoprefixer/extension.rb:74:in `process'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/middleman-autoprefixer-2.10.1/lib/middleman-autoprefixer/extension.rb:60:in `call'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/rack-2.2.4/lib/rack/head.rb:12:in `call'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/rack-2.2.4/lib/rack/lint.rb:50:in `_call'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/rack-2.2.4/lib/rack/lint.rb:38:in `call'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:244:in `call'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/rack-2.2.4/lib/rack/handler/webrick.rb:95:in `service'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/webrick-1.7.0/lib/webrick/httpserver.rb:140:in `service'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/webrick-1.7.0/lib/webrick/httpserver.rb:96:in `run'
	/home/HENRIQUE.TEIXEIRA/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/webrick-1.7.0/lib/webrick/server.rb:310:in `block in start_thread'
```

### CODE REVIEW

**Comment by henrique-ft** [comment]
It still dont work in `2.8.1`

**Comment by tnir** [comment]
@henrique-ft It looks like you need to do `bundle && git add Gemfile.lock && git commit --amend` to pass the CI pipeline.

**Review by tnir — COMMENTED** [review]
I think you should modify https://github.com/middleman/middleman/blob/5902a84004f6aa1b023cbb029ae1cc04e78e770a/middleman-core/middleman-core.gemspec#L52 instead.

File: Gemfile [hunk 1]
```diff
@@ -41,4 +41,7 @@ gem 'simplecov', '~> 0.21', require: false
 gem 'middleman-cli', path: 'middleman-cli'
 gem 'middleman-core', path: 'middleman-core'
 
+# Locked in 2.7.0 until https://github.com/rails/execjs/issues/99 be solved (still dont work in 2.8.1)
+gem 'execjs', '2.7.0'
```
Comment (markets) [inline Gemfile:45]: hi @henrique-ft 👋🏼  

This is not necessarya anymore ✂️, as it was already defined in the gemspec now

**Review by markets — APPROVED** [review]

**Pull request merged**